map/open-mobile-map-package/README.md

```diff
@@ -2,7 +2,7 @@
 
 Display a map from a mobile map package.
 
-![Image of open mobile map package](OpenMobileMapPackage.png)
+![](OpenMobileMapPackage.png)
```

Comment:
missing alt text: we should keep these in for the website since all these alt text tags were agreed on by the samples and accessibility teams at Esri.